### PR TITLE
README - Fix typos

### LINKED ISSUE #3023: Quadrillion vs. Quintillion
https://github.com/ripple/rippled/blob/develop/README.md needs to be updated. More specifically "Key Features of the XRP Ledger" section so that is in line with https://xrpl.org/xrp-ledger-overview.html#the-digital-asset-for-payments
There is/was grand total of 100 quadrillion (10^17) drops of XRP. Not 100 quintillion drops which would give us 10 trillion XRPs but hey who's counting ;).

### LINKED ISSUE #3026: Readme.md: minor wording correction
_## XRP
XRP is a public, counterparty-less asset native to the XRP Ledger, and is designed to bridge the many different currencies in use worldwide. XRP is traded on the open-market and is available for anyone to access. The XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP. Its creators gifted 80 billion XRP to a company, now called [Ripple](https://ripple.com/), to develop the XRP Ledger and its ecosystem.  Ripple uses XRP the help build the Internet of Value, ushering in a world in which money moves as fast and efficiently as information does today._

Last sentence in second paragraph of README.md should be changed from "Ripple uses XRP the help build the Internet of Value, ushering in a world in which money moves as fast and efficiently as information does today." to "Ripple uses XRP to help build the Internet of Value, ushering in a world in which money moves as fast and efficiently as information does today."

### PR DESCRIPTION
**Pull request by intelliot**
Fix #3026 - thanks @William-Gomez
- `the` -> `to`

Fix #3023 - thanks @crypto-deus
- `quintillion` -> `quadrillion`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The server software that powers the XRP Ledger is called `rippled` and is availa
...
 - **[Fast, Efficient Consensus Algorithm][]:** The XRP Ledger's consensus algorithm settles transactions in 4 to 5 seconds, processing at a throughput of up to 1500 transactions per second. These properties put XRP at least an order of magnitude ahead of other top digital assets.
-- **[Finite XRP Supply][]:** When the XRP Ledger began, 100 billion XRP were created, and no more XRP will ever be created. (Each XRP is subdivisible down to 6 decimal places, for a grand total of 100 quintillion _drops_ of XRP.) The available supply of XRP decreases slowly over time as small amounts are destroyed to pay transaction costs.
+- **[Finite XRP Supply][]:** When the XRP Ledger began, 100 billion XRP were created, and no more XRP will ever be created. (Each XRP is subdivisible down to 6 decimal places, for a grand total of 100 quadrillion _drops_ of XRP.) The available supply of XRP decreases slowly over time as small amounts are destroyed to pay transaction costs.
```
Comment (nbougalis) [inline README.md]: A couple of comments:

>Each XRP is subdivisible down to 6 decimal places, for a grand total of 100 quadrillion _drops_ of XRP.

I don't like this wording. A _drop_ is a unit the same way a cent is. You'd never say: "I withdrew 100,000 cents of dollar."

We should have something like this instead:

>Each XRP is subdivisible to 1,000,000 _drops_ for a grand total of 100 quadrillion _drops_ of XRP.

Furthermore, it occurs to me that this parenthetical just confuses things unnecessarily and in this context may not anything to the conversation. What does it matter how many quadrillion drops there are? The supply of XRP is accounted for in XRP in every place where a human is liable to look (except in the ledger internals, and I think that distinction belongs in the documentation and not the general README file).

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
 
 ## XRP
-XRP is a public, counterparty-less asset native to the XRP Ledger, and is designed to bridge the many different currencies in use worldwide. XRP is traded on the open-market and is available for anyone to access. The XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP. Its creators gifted 80 billion XRP to a company, now called [Ripple](https://ripple.com/), to develop the XRP Ledger and its ecosystem.  Ripple uses XRP the help build the Internet of Value, ushering in a world in which money moves as fast and efficiently as information does today.
+XRP is a public, counterparty-less asset native to the XRP Ledger, and is designed to bridge the many different currencies in use worldwide. XRP is traded on the open-market and is available for anyone to access. The XRP Ledger was created in 2012 with a finite supply of 100 billion units of XRP. Its creators gifted 80 billion XRP to a company, now called [Ripple](https://ripple.com/), to develop the XRP Ledger and its ecosystem.  Ripple uses XRP to help build the Internet of Value, ushering in a world in which money moves as fast and efficiently as information does today.
```
Comment (nbougalis) [inline README.md]: Is "counterparty-free" better? Or perhaps something more extensive:

>XRP is the public digital asset, native to the XRP Ledger. Designed as a fast and efficient bridge for the many different currencies in use worldwide, XRP is free of counterparty risk and available for anyone to use. The XRP Ledger ws created [...]

**Comment by intelliot** [comment]
@mDuo13 Let me know if you'd like to adopt the above changes, and I can put them in.

File: README.md (hunk 1 above)
Comment (mDuo13) [reply README.md]: That's fair. I would be fine with removing the parenthetical entirely here.

File: README.md (hunk 2 above)
Comment (mDuo13) [reply README.md]: Would be fine with "counterparty-free" here. I think I prefer the original wording to Nik's longer rewrite (sorry, @nbougalis!) though.

Comment (nbougalis) [reply README.md]: No need to apologize. I've been accused of being many things; laconic isn't one of them.

**Pull request closed without merging**